### DISCUSSION TITLE
Fedora 38 + Artisan 2.8.4

### DISCUSSION THREAD

**Discussion by phobo74** [artisan-roaster-scope/artisan#1263] [General]
Hello !

I am trying to install Artisan on Fedora 38 (rpm -i/dnf install or AppImage). However at launch nothing happens...
The process exits. Any ideas or clues ? Is there any log directory to look into ?
I have added the user to the dialout group.

Thanks for your help !

**Comment by tomhusband** [comment]
Sorry, I wish I could help. I actually settled on Linux Mint but that shouldn't make a difference. Have you asked Marko Luther? He's been very helpful to me in the past.
